### PR TITLE
Add compat data for SVGZoomAndPan

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/SVGZoomAndPan

### CODE REVIEW

File: api/SVGZoomAndPan.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGZoomAndPan",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/SVGZoomAndPan.json]: All 3 versions of Chrome should be `true`. https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/svg/svg_zoom_and_pan.idl

File: api/SVGZoomAndPan.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/SVGZoomAndPan.json]: Both versions of Opera should be true.

File: api/SVGZoomAndPan.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "SVGZoomAndPan": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGZoomAndPan",
+        "support": {
+          "webview_android": {
```
Comment (connorshea) [inline api/SVGZoomAndPan.json]: @maboa please put `webview_android` at the end of the browser list to keep it alphabetical, see #1882. Unfortunately this'll need to be fixed on all your PRs 😬

**Comment by maboa** [comment]
Changes pushed :)

**Pull request merged**